tests/filecheck/dialects/smt/ops.mlir

```diff
@@ -1,31 +1,44 @@
 // RUN: XDSL_ROUNDTRIP
 
+%const1 = smt.declare_fun : !smt.bool
```

Comment:
I thought we were trying to prefer check statements before code? (Realise this is not followed everywhere)

Reply:
I changed it in the entire file